### PR TITLE
Migrated recipe to be executed on v7

### PR DESCRIPTION
**Pull request by rupikakapoor**

### CODE REVIEW

**Review by mushyy — CHANGES_REQUESTED** [review]
Needs to be ran through linting process as per our [CONTRIBUTING.md](https://github.com/engineyard/ey-cookbooks-dev-v7/blob/master/CONTRIBUTING.md) guidelines

**Comment by rupikakapoor** [comment]
@mushyy: indentation fix is implemented:

```
rupika@Rupikas-iMac ey-cookbooks-dev-v7 % cookstyle               
Inspecting 190 files
..............................................................................................................................................................................................

190 files inspected, no offenses detected
```

File: cookbooks/ey-nodejs/attributes/default.rb [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-                            '8.16.2'
-                          when 'nodejs_6'
-                            '6.17.1'
-                          when 'nodejs_5'
-                            '5.12.0'
-                          when 'nodejs_4'
-                            '4.9.1'
+                          when "nodejs_14"
+                            "14.19.3"
+                          when "nodejs_15"
```
Comment (mushyy) [inline cookbooks/ey-nodejs/attributes/default.rb:12]: Wrong key for NodeJS should be nodejs_16

**Review by mushyy — CHANGES_REQUESTED** [review]
Other than the requested change this looks good to me

**Comment by rupikakapoor** [comment]
@mushyy fixed

**Comment by mushyy** [comment]
Looks good to me ready for QA.

**Pull request merged**